fintech-examples/fintech-ui/src/app/bank-search/common/search/search.component.html

```diff
@@ -2,6 +2,7 @@
   <i class="fa fa-search form-control-feedback"></i>
   <input
     [formControl]="searchTerm"
+    [value]="'adorsys'"
```

Comment:
why. do we bind the form value to 'adorsys'?

Reply:
weil das den Test wahnsinnig beschleunigt. Und weil es derzeit die einzige Bank ist, mit der überhaupt was funktioniert. Das können wir später dann rauswerfen. Ich markiere es mit TODO.

Reply:
ok